### DISCUSSION TITLE
Copy Error Messages

### DISCUSSION THREAD

**Discussion by HunterJacobs** [burkybang/SuiteAdvanced-Feedback#135] [Ideas]
Currently I am not able to copy the text on a netsuite error message. This would be very helpful as I am always using chatgpt to help resolve my problems. Example of error message below:
![image](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/6316084/b91506c5-a2d8-4095-a127-58cefa25ae94)

**Comment by burkybang** [comment]
Are you sure you can't copy it? That's a browser alert box. I'm able to copy text from alert boxes just fine. I'm using Chrome.

![image](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/927886/5defdcf4-4d15-4254-8419-14464ee2aa99)

**Comment by HunterJacobs** [comment]
It looks like this is coming from the Netsuite Freemarker error log - maybe a bit more obscure than I originally thought
